.github/workflows/CI.yaml

```diff
@@ -29,9 +29,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.9", "3.10", "3.11"]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
@IAlibay opinion on keeping vs removing 3.9 support?

Reply:
Doesn't hurt for now.